sdk/webpubsub/azure-messaging-webpubsubservice/tests/test_jwt.py

```diff
@@ -117,3 +117,22 @@ def test_pass_in_jwt_headers(connection_string):
     kid = '1234567890'
     token = client.get_client_access_token(jwt_headers={"kid":kid })['token']
     assert jwt.get_unverified_header(token)['kid'] == kid
+
+test_cases = [
+    ("Endpoint=http://localhost;Port=8080;AccessKey={};Version=1.0;".format(access_key), "hub", "ws://localhost:8080/clients/mqtt/hubs/hub"),
+    ("Endpoint=https://a;AccessKey={};Version=1.0;".format(access_key), "hub", "wss://a/clients/mqtt/hubs/hub"),
+    ("Endpoint=http://a;AccessKey={};Version=1.0;".format(access_key), "hub", "ws://a/clients/mqtt/hubs/hub")
+]
+@pytest.mark.parametrize("connection_string,hub,expected_url", test_cases)
```

Comment:
I would expect more test cases for something that does url manipulation and parsing....

Reply:
Sure @johanste. There are actually already other test cases for this method `get_client_access_token` within this file `test_jwt_py` already. I'm just adding more cases relating to the new MQTT token parser logic